test/test_context.py

```diff
@@ -934,6 +934,10 @@ def test_data_boundaries(self):
         meta_cols = set(('geom_id', 'geom_tags', 'geom_type', ))
         self.assertTrue(meta_cols & set(boundary_meta.columns))
 
+        # boundary metadata with correct timespan
+        meta_2015 = cc.data_boundaries(timespan='2015')
+        self.assertTrue(meta_2015[meta_2015.valid_timespan].shape < boundary_meta.shape)
```

Comment:
How about making this test more explicit, like
1. we know there are boundaries for 2015, we could do `self.assertTrue(meta[meta.valid_timespan].shape[0] > 0)`
2. test for the year 9999 and assert that the number of entries is zero